app/presenters/image_upload_response_presenter.rb

```diff
@@ -48,6 +49,10 @@ def url_options
 
   private
 
+  def doc_auth_result_failed?
+    @form_response.to_h[:doc_auth_result] == 'Failed'
```

Comment:
For reference:

https://github.com/18F/identity-idp/blob/46c44f70244efee4aa77cc48df1312058b3257eb/app/services/doc_auth/acuant/result_codes.rb#L8-L19

Reply:
Oh, that's a good idea! (We don't conventionally do that in other places, but I think it makes sense to avoid the magic strings)

Reply:
Updated in d165e88.